.env.example

```
@@ -0,0 +1,19 @@
+# Bot Configuration
+BOT_TOKEN=your_bot_token_here
+DEV_ID=your_user_id_here
+
+# Optional: MongoDB connection (defaults to SQLite if left empty)
+DB_URL=
+
+# API and Web Management
+WEBUI_PORT=3000
+API_KEY=your_secure_api_key_here
+# Comma-separated list of origins allowed to access the API
+ALLOWED_ORIGINS=https://aometry.finneh.xyz,http://localhost:4321
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALLOWED_ORIGINS key should go before the API_KEY key

(UnorderedKey)